src/main/resources/db/changelog/changeset/01-create-table-books.yaml

```diff
@@ -0,0 +1,42 @@
+databaseChangeLog:
```

Comment:
I do not think we have liquibase addition as part of the tasks, and also you have it disabled in the `app.properties`, so I'd remove that and other stuff that is related to that for now because it is not used

Reply:
Since the task specifies that you need to use the create-drop parameter, I have disabled the use of Liquibase for now. But I would like to keep it for the future and not delete it.